File: python/2017/4/solution1.py
with open("../../../data/2017/4/data.txt", "rb") as f:
    lines = [line.replace('\n','').split() for line in f.readlines()]

count = 0
for line in lines:
    if len(set(line)) == len(line):
        count += 1

print "Solution is %i" % count

File: julia/2017/4/solution1.jl
f = open("../../../data/2017/4/data.txt")
lines = readlines(f)
close(f)

count = 0
for line in lines
    line = split(line, " ")
    if length(line) == length(Set(line))
        count += 1
    end
end

println("Solution is $count")
